src/command/addIdentityToUser.ts

```diff
@@ -0,0 +1,35 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import { IdentityCommandProps } from "../@types/IdentityCommandProps";
+import { fetchManagement } from "../utils/fetchManagement";
+
+const ADD_IDENTITY_TO_USER = gql`
+  mutation addIdentityToUser(
+    $userId: String!
+    $type: IdentityTypes!
+    $value: String!
+  ) {
+    addIdentityToUser(input: { userId: $userId, type: $type, value: $value }) {
+      id
+      primary
+      status
+      type
+      value
+      __typename
```

Comment:
curiosity, why is that needed?

Reply:
For now, none of these are need (even the ID, as we don't use apollo caching). Do you want me to remove them?